### PR TITLE
[BEAM-3975] Upgrade virtualenv and tox in Jenkins machines.

### PR DESCRIPTION
**Pull request by udim**
DESCRIPTION HERE

------------------------

Follow this checklist to help us incorporate your contribution quickly and easily:

 - [ ] Make sure there is a [JIRA issue](https://issues.apache.org/jira/projects/BEAM/issues/) filed for the change (usually before you start working on it).  Trivial changes like typos do not require a JIRA issue.  Your pull request should address just this issue, without pulling in other changes.
 - [ ] Format the pull request title like `[BEAM-XXX] Fixes bug in ApproximateQuantiles`, where you replace `BEAM-XXX` with the appropriate JIRA issue.
 - [ ] Write a pull request description that is detailed enough to understand:
   - [ ] What the pull request does
   - [ ] Why it does it
   - [ ] How it does it
   - [ ] Why this approach
 - [ ] Each commit in the pull request should have a meaningful subject line and body.
 - [ ] Run `mvn clean verify` to make sure basic checks pass. A more thorough check will be performed on your pull request automatically.
 - [ ] If this contribution is large, please file an Apache [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf).

### CODE REVIEW

**Comment by udim** [comment]
run python postcommit

**Comment by udim** [comment]
R: @aaltay

**Comment by udim** [comment]
Note to reviewer: PostCommit is failing (after this PR) due to https://issues.apache.org/jira/browse/BEAM-3922 (or possibly another bug I'm not aware of).
The issue fixed here is that the Tox version was too old for the `extras` flags I add in a previous PR.
http://tox.readthedocs.io/en/latest/config.html#confval-extras=MULTI-LINE-LIST

File: sdks/python/run_postcommit.sh [hunk 1]
```diff
@@ -32,16 +32,14 @@ set -v
...
-pip install virtualenv --user
-
-# INFRA does not install tox
-pip install tox --user
+# INFRA does not install these packages
+pip install --user --upgrade virtualenv tox
 
 # Tox runs unit tests in a virtual environment
-${LOCAL_PATH}/tox -e ALL -c sdks/python/tox.ini
+${LOCAL_PATH}/tox -e ALL -c sdks/python/tox.ini -v -v
```
Comment (aaltay) [inline sdks/python/run_postcommit.sh]: What does `-v -v` do?

**Pull request merged**